### PR TITLE
docs: add section to migrate from eslint-loader

### PR DESCRIPTION
**Pull request by ricardogobbosouza**
Add a short description of migration from eslint-loader to this plugin.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,6 +26,43 @@ The plugin was born with the purpose of solving some problems of the [eslint-loa
 | Using cache directly from eslint |  :heavy_check_mark:   | :heavy_multiplication_x: |
 | Lint only changed files          |  :heavy_check_mark:   | :heavy_multiplication_x: |
 
+## Migrate from `eslint-loader`
+
+The loader `eslint-loader` will be deprecated soon, please use this plugin instead.
```
Comment (evilebottnawi) [inline README.md:31]: Can we move it on top of the file? And using `**DEPRECATION**`?

Comment (ricardogobbosouza) [reply README.md:31]: `**DEPRECATION**` would be in the `eslint-loader,` right?

Comment (ricardogobbosouza) [reply README.md:31]: Here in `eslint-webpack-plugin` it would be just a migration guide

Comment (ricardogobbosouza) [reply README.md:31]: Should this migration guide be on the `eslint-loader` only?

Comment (evilebottnawi) [reply README.md:31]: Oh, yes, my mistake :smile: All fine

> Should this migration guide be on the eslint-loader only?

On both repo for better undestanding

**Pull request merged**